App.tsx

```diff
@@ -115,13 +117,15 @@ export default function App() {
                     : MaterialLightTheme
                 }
               >
-                <PortalProvider>
-                  <View style={styles.safe}>
-                    <XmtpEngine />
-                    <Main />
-                    <DebugButton ref={debugRef} />
-                  </View>
-                </PortalProvider>
+                <GestureHandlerRootView style={{ flex: 1 }}>
```

Comment:
Want to include this from a stylesheet?

Reply:
@alexrisch The reason I didn't do it (and also didn't in a lot of other places) is because I feel we should talk about how we want to handle styles in the app. There are many ways. For example, I've always put the styles inline. I feel it's much faster to develop that way. And usually if we do it right, we have amazing composability so there are not that many inline styles that pollutes everything

Reply:
@thierryskoda doesn't this pose rerender issues?
Isn't each style object then seen as a new object?

Reply:
Yes it will rerender you're right! Often my strategy tho is not worry about those because this component shouldn't rerender anyway.

But happy to make it clean and put it in memoized style object instead. Just need to figure out the way we'll do this. (useStyle?, createStyleSheet? objects like Ignite is doing? Tailwind class style?).